src/api.ts

```diff
@@ -1,7 +1,7 @@
 import { MovieData } from './types/MovieData';
 import { ResponseError } from './types/ReponseError';
 
-const API_URL = 'https://www.omdbapi.com/?apikey=your-key';
+const API_URL = 'http://www.omdbapi.com/?i=tt3896198&apikey=7a42afed';
```

Comment:
```suggestion
const API_URL = 'https://www.omdbapi.com/?apikey=7a42afed;';

```

I have an error trying to find a movie, try to rewrite your key in this way and test if it works